aspy/refactor_imports/classify.py

```diff
@@ -97,6 +97,7 @@ def _get_module_info(
 def classify_import(
         module_name: str,
         application_directories: Tuple[str, ...] = ('.',),
+        application_modules: Tuple[str, ...] = (),
```

Comment:
hmmm, I think this argument should sound slightly more scary -- as it's written right now it sounds like I would always want to use this but really it's only for very very special cases -- I don't exactly have any suggestions though 🤔 

the type here could also be relaxed a bit, I believe this could be `Container[str]`?

Reply:
I went with `override_absent_application_modules` as it sounds a bit like you shouldn't be doing it. It is a bit too long, but that does add to the scariness.

Reply:
cool, I like it -- I'll think a little bit more about alternatives as well before merging 👍